sway/input/seatop_default.c

```diff
@@ -642,7 +645,7 @@ static void handle_tablet_tool_motion(struct sway_seat *seat,
  * Functions used by handle_pointer_axis  /
  *--------------------------------------*/
 
-static uint32_t wl_axis_to_button(struct wlr_event_pointer_axis *event) {
+static uint32_t axis_to_button(struct wlr_event_pointer_axis *event) {
```

Comment:
Ehh, I wouldn't worry about the functions in swaybar like `wl_axis_to_button`, because "wl_axis" is an actual enum from the wayland protocol, so a function that translates a "wl_axis" value to something is appropriately named "wl_axis_to_something", even though it's not a symbol from libwayland. Similar for the wl_pointer object below.

On the contrary a "swipe_button" isn't anything related to the wayland protocol so it's just not appropriate here. I think just drop this commit; it's not necessary.

Reply:
That explains it. Dropped.